src/gettingstarted/local-development/build-locally.md

```diff
@@ -0,0 +1,47 @@
+# Local development
+
+## Build site locally
+
+Now that you've opened tunnels into your services, you'll have access to all of your data in your environment. All that's left now is to actually build the site.
+
+<asciinema-player src="/videos/asciinema/build.cast" preload=1 autoplay=1 loop=1></asciinema-player>
+
+1. **Build the site**
+
+    From the repository root, run the command
+
+    ```bash
+    platform build
+    ```
```

Comment:
I really hate this command, because it doesn't like you do true live edits except sometimes maybe if it happens to work out.  I can see this being potentially very confusing.  Should we expand more on how _www uses symlinks and such so it sometimes does live editing and sometimes doesn't?  (Really, this command was built for PHP and may not even work for anything else.)

Reply:
Tried to address this point, let me know if this works for you.